Formula/libpostal.rb

```diff
@@ -0,0 +1,28 @@
+class Libpostal < Formula
+  desc "C library for parsing/normalizing street addresses around the world"
+  homepage "https://github.com/openvenues/libpostal"
+  url "https://github.com/openvenues/libpostal/archive/v1.0.0.tar.gz"
+  sha256 "3035af7e15b2894069753975d953fa15a86d968103913dbf8ce4b8aa26231644"
+  license "MIT"
+  head "https://github.com/openvenues/libpostal.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "curl" => :build
```

Comment:
can it not work with the macOS version?

Reply:
Good question; the project docs say to install it with brew first so I added it as a dependency. I can confirm though.